src/XR/features/WebXRControllerPointerSelection.ts

```diff
@@ -558,6 +659,34 @@ export class WebXRControllerPointerSelection extends WebXRAbstractFeature {
                         }
                     }
                 });
+
+                // squeezeComponent is a new member on the controllerData
+                controllerData.squeezeComponent = motionController.getComponent("grasp");

```

Comment:
Is it possible that this is null on certain controllers?  If it can be null, the next line will fail.